server/api/db/sqldb/db.py

```diff
@@ -2496,7 +2496,9 @@ def tag_objects_v2(
         self._upsert(session, tags)
 
     # ---- Projects ----
-    def create_project(self, session: Session, project: mlrun.common.schemas.Project):
+    def create_project(
+        self, session: Session, project: mlrun.common.schemas.Project, with_summary=True
+    ):
```

Comment:
Not sure it's worth adding this complication to the logic, to appease a unit test. We will only ever use this flag for the test but if it gets used by mistake it could cause issues.
Instead, maybe create the project record directly in the test without calling this method. Or separate the creation of the summary to a different method, and the UT can monkey patch it to do nothing.